src/config/index.ts

```diff
@@ -183,7 +183,7 @@ export async function set<T extends SchemaTypeKey>(
 	if (trx != null) {
 		await setValuesInTransaction(trx);
 	} else {
-		await db.transaction((tx: Transaction) => setValuesInTransaction(tx));
```

Comment:
I removed the unnecessary explicit type so we can automatically use the correct types implicitly